docs/content/exporting/http/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: HTTP
```

Comment:
Perhaps the title here should be HTTP/HTTPS now to make it more clear that it covers both?